bigbluebutton-html5/imports/api/2.0/users/server/modifiers/userKicked.js

```diff
@@ -0,0 +1,38 @@
+import { check } from 'meteor/check';
+import Logger from '/imports/startup/server/logger';
+import Users from '/imports/api/2.0/users';
+
+export default function kickUser(meetingId, userId) {
+  check(meetingId, String);
+  check(userId, String);
+
+  const selector = {
+    meetingId,
+    userId,
+  };
+
+  const modifier = {
+    $set: {
```

Comment:
We don't use a nested `user` key anymore.

Reply:
... and as far as I can see - voiceUser is in another collection now.